optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -843,8 +843,8 @@
 
       <para>Requires the score calculation to support <link linkend="explainingTheScore">constraint matches</link>.
       <link linkend="droolsScoreCalculation">Drools score calculation</link> supports constraint matches automatically,
-      but <link linkend="incrementalJavaScoreCalculation">incremental Java score calculation</link> requires requires
-      more work.</para>
+      but <link linkend="incrementalJavaScoreCalculation">incremental Java score calculation</link> requires more work.
+      </para>
```

Comment:
Just my $0.02: the `</para>` elements should probably (for consistency) be put at the end of the previous line (see below for more occurrences)